llama-index-core/llama_index/core/storage/docstore/keyval_docstore.py

```diff
@@ -206,7 +206,7 @@ def _prepare_kv_pairs(
 
     def add_documents(
         self,
-        nodes: Sequence[BaseNode],
+        docs: Sequence[BaseNode],
```

Comment:
why rename this? Technically it is nodes 😅

Reply:
> why rename this? Technically it is nodes 😅

I agree with you 100%. The problem I encountered was wherever the parameter is typed as the base class, if we provide what the typing information says: `docs: Sequence[BaseNode],` you will get a nasty error where your kwarg was named `docs`, but you provided `nodes`.

```py
@abstractmethod
    def add_documents(
        self,
        docs: Sequence[BaseNode],
        allow_update: bool = True,
        batch_size: int = DEFAULT_BATCH_SIZE,
        store_text: bool = True,
    ) -> None:
        ...
```

Reply:
right, but its a positional argument, so it doesn't need to be provided as a kwarg?

Reply:
`add_documents(nodes, ...)` should work fine no?

Reply:
Correct, but if somebody does try to provide it as a kwarg they will get an error that can be difficult to figure out. I'm happy to change the base class to nodes and try to find everywhere else that docs is used.